lib/sync/util.js

```diff
@@ -7,7 +7,7 @@ var _ = require('underscore');
 var SYNC_LOGGER = 'SYNC';
 var loggers = {};
 
-var generateHash = function (plainText) {
+function generateHash(plainText) {
```

Comment:
@wei-lee thinking about this more, do we have any guidelines (or are there any existing 3rd party guidelines that we can refer to), on when to use one over the other? AIUI the main difference is scoping, so either should be fine in this case.

There are other places in this module where function declarations are used in cases like this (such as in `lib/sync/index.js`). I think it would be good to have a consistent way of knowing which to use where (like "use X instead of Y, except in these specific circumstances").

Unless we have such a thing already, and I'm just not aware of it! :)